### ISSUE TITLE
Vim text objects

### ISSUE THREAD

**Issue by ritajitk** [vim-latex/vim-latex#155]
Most of vim's power lies in its **text objects**. To change the text inside '{ }' I'll press '**ci{'** (change in { )  and text in between the braces will be removed. This kind of vim actions makes editing very easy. 
So, this is more of an enhancement request:
It'll be really nice to have latex environments, commands, sections, $ as vim text objects, i.e. If I wanted to change the text inside an environment then  it'll be done through '**cie**' (change in environment) or something similar. It'll make editing more efficient. 

Thanks for making this plugin.
Made life much easier.

**Comment by gerw** [comment]
You are right, text objects are a great feature in vim. I already use text objects for editing tex files, see https://github.com/gerw/vim-conf-gerw/blob/master/ftplugin/tex.vim#L223. However, I did not had time to include these into vim-latex suite. Feel free to use my code and file a pull request!

**Comment by ritajitk** [comment]
Awesome! the ftplugin is very nice. I'll try to file a pull request. Thanks!

**Comment by kiryph** [comment]
You can also use the text objects from [vimtex](https://github.com/lervag/vimtex/blob/master/doc/vimtex.txt#L158-L163) and disable all other features of vimtex (folding, compilation, ....) where you use vim-latex.
